### ISSUE TITLE
Tailwind is throwing an error in production (not dev)

### ISSUE THREAD

**Issue by bukinoshita** [vinicoder/tw-to-css#2]
Given the following simple email:

```ts
<Html>
    <Head />
    <Tailwind>
      <Button className="bg-slate-900 text-slate-900">Test</Button>
    </Tailwind>
  </Html>
```

When run in production the following error is thrown:

```
TypeError: e.replaceAll is not a function
    at dp (/opt/render/project/src/node_modules/tw-to-css/dist/index.js:41:516)
    at /opt/render/project/src/node_modules/tw-to-css/dist/index.js:41:406
    at Array.map (<anonymous>)
    at dp (/opt/render/project/src/node_modules/tw-to-css/dist/index.js:41:399)
    at /opt/render/project/src/node_modules/tw-to-css/dist/index.js:41:567
    at /opt/render/project/src/node_modules/tw-to-css/dist/index.js:41:779
    at replaceTailwindStyles (/opt/render/project/src/node_modules/@react-email/tailwind/dist/index.js:50:41)
    at /opt/render/project/src/node_modules/react/cjs/react.production.min.js:18:85
    at R (/opt/render/project/src/node_modules/react/cjs/react.production.min.js:16:230)
    at Object.S [as map] (/opt/render/project/src/node_modules/react/cjs/react.production.min.js:18:52)
```

It looks like `classList` on this line of `tw-to-css` is undefined/null: https://github.com/vinicoder/tw-to-css/blob/c98d9b009c4cec298846eed4bb53f976675d5197/src/index.ts#L43

Note that this doesn't happen in development, only production.

---

Reported on https://github.com/resendlabs/react-email/issues/506

**Comment by ericallam** [comment]
This error is occurring because Node.js <= 14 does not support the `String.replaceAll` function, and the reason it would only happen in production is because the production Node.js version is below version 15 (in this case, the code was deployed to Render.com, which has a default Node.js version of 14.17.0)

**Comment by vinicoder** [comment]
Hey @ericallam! That makes sense. Thank you for reporting and bringing more details. I will work on it to have more compatibility with past node versions.

**Comment by vinicoder** [comment]
Hey everyone! I just fixed it on release 0.0.11
https://github.com/vinicoder/tw-to-css/blob/main/src/index.ts#L43

Also, I created a PR on `react-email`: https://github.com/resendlabs/react-email/pull/528
